_posts/2018-04-02-taldoor.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: TalDoor aka "GateKeeper V2"
+date: 2018-04-02
+categories:
+  - projects
+description: A small POE powered door lock system using the PocketBeagle along with a PN532 NFC module to scan and authenticate user ID cards and unlock the door.
+image: https://raw.githubusercontent.com/akaStanley/TalDoor/master/Board%20Photos/BothBoards_angle.PNG
+image-sm: https://raw.githubusercontent.com/akaStanley/TalDoor/master/Board%20Photos/BothBoards_angle.PNG
+author: Spencer Kulbacki
+author-image: https://avatars2.githubusercontent.com/u/14046811?s=460&v=4
+author-bio: 4th year Electrical Engineering Technology major and Research & Development Director 2017-18.
+author-email: sxk4788@rit.edu
+author-social:
+  github: https://github.com/akaStanley
+  linkedin: https://www.linkedin.com/in/skulbacki/
+---
+TalDoor is a hardware redesign of CSH's famous (now severely outdated and unsupported) iButton authentication system. TalDoor is named after our illustrious RA Tal, who was famous for having his door closed at all hours. TalDoor is a complete redesign of the original Gatekeeper system, with the main constraint of maintaining the same-footprint. 
+
+The two main complaints about the old GateKeeper system are that iButtons are insecure and finding custom power supplies for the hardware is difficult and costly. So when making TalDoor I chose to use the [PocketBeagle]( http://beagleboard.org/pocket) devboard. In addition to often returning pictures of cute dogs when googling the name, the PocketBeagle runs linux and is powerful enough to support encryption of the ID card data.
```

Comment:
`s/devboard/development board/`
`s/googling/Googling/`
`s/linux/Linux/`
Maybe try "encrypted ID cards" instead of "encryption of the ID card data"?